src/app/jscaip/AI/AlignmentHeuristic.ts

```diff
@@ -1,14 +1,52 @@
-import { MGPOptional } from '@everyboard/lib';
+import { MGPOptional, Utils } from '@everyboard/lib';
 import { Heuristic } from './Minimax';
 import { EmptyRulesConfig, RulesConfig } from '../RulesConfigUtil';
 import { BoardValue } from './BoardValue';
-import { SCORE } from '../SCORE';
 import { Coord } from '../Coord';
 import { Move } from '../Move';
 import { GameState } from '../state/GameState';
+import { Player } from '../Player';
+import { GameStatus } from '../GameStatus';
+
+/**
+ * Represents possible alignment configurations
+ **/
+export class AlignmentStatus {
+
+    // There's nothing special about this alignment
```

Comment:
its then actually not an alignement right ?

Reply:
it can be an alignment of n-2 !
mais je change à "there's no particular alignment"